.github/workflows/reusable-helmfile-deploy.yaml

```
@@ -0,0 +1,44 @@
+---
+name: Deploy Helmfile
+on:
+  workflow_call:
+    inputs:
+      ENVIRONMENT:
+        required: false
```

```suggestion
        required: true
```

Because of `-e ${{ inputs.ENVIRONMENT }} sync`.